docs/checkers/index.md

```diff
@@ -1740,6 +1740,26 @@ class CashRegister:
 
 ## Exceptions
 
+(W0719)=
+
+### Broad exception raised (W0719)
+
+This error is emitted when one raises a generic exception. Raising exceptions that are not specific will cause the
+program to catch generic exceptions. This is bad practice because we may catch exceptions that we don't want.
+Catching generic exceptions can also hide bugs and make it harder to debug programs.
+
+```{literalinclude} /../examples/pylint/w0702_bare_except.py
```

Comment:
This file is not correct. You can build the documentation (see README for instructions) on your computer and double-check that your changes show up appropriately. :)